docs/sources/mimir/release-notes/v2.7.md

```diff
@@ -0,0 +1,81 @@
+weight: 300
+---
+
+# Grafana Mimir version 2.7 release notes
+
+Grafana Labs is excited to announce version 2.7 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Store-gateway streaming enabled by default** The new default value of `5000` for `-blocks-storage.bucket-store.batch-series-size` enables store-gateway streaming in the default configuration. This means that series are loaded from object storage in batches rather than buffering them all in memory before returning to the querier. Enabling streaming can reduce memory utilization peaks in the store-gateway.
+- **Store-gateway index header reader no longer uses mmap by default** Along with streaming enabled in the store-gateway, this change contributes to more efficient memory usage. See the **Important changes** section for more details.
+- **Support for `keep_firing_for` option to ruler configuration** This new option determines the amount of time an alert should keep firing while the ruler expression doesn't return results.
+- **More efficient chunks fetching and caching** Enable with the new experimental feature flag `-blocks-storage.bucket-store.chunks-cache.fine-grained-chunks-caching-enabled=true`. This should reduce CPU, memory utilization, and receive bandwidth of a store-gateway.
+- **Experimental query sharding improvements**:
+  A new configuration parameter, `-query-frontend.query-sharding-target-series-per-shard`, allows query sharding to take into account cardinality of similar requests executed previously when computing the maximum number of shards to use. If you want to try it out, we recommend starting with a value of `2500`.
+- **Experimental support for native histogram ingestion**:
+  Native histograms can now be ingested. The new per-tenant limit `-ingester.native-histograms-ingestion-enabled` controls whether native histograms are stored or ignored. The support for querying native histograms is not complete yet and it's expected to be available in the next release.
+
+### Alertmanager improvements
+
+- **New metrics** The following upstream metrics are now exposed:
+  - `cortex_alertmanager_dispatcher_aggregation_groups`
+  - `cortex_alertmanager_dispatcher_alert_processing_duration_seconds`
+
+### Helm chart improvements
+
+The Grafana Mimir and Grafana Enterprise Metrics Helm chart is now released independently. See the [Grafana Mimir Helm chart documentation](/docs/helm-charts/mimir-distributed/latest/).
+
+## Important changes
+
+In Grafana Mimir 2.7, the default vaules of the following configuration options have changed:
+
+- `-blocks-storage.bucket-store.batch-series-size` is now enabled by default with a value of `5000`.
+- `-blocks-storage.tsdb.retention-period` has changed from `24h` to `13h`.
```

Comment:
This change comes from PR #4382, which isn't part of this release. We can either cherry-pick it, or remove this line.

Reply:
Thanks, I removed it